### DISCUSSION TITLE
ISTIO installation with modified Helm charts

### DISCUSSION THREAD

**Discussion by Josephred999** [istio/istio#45891] [Q&A] [answered]
Hello, 
I am trying to setup ISTIO via Helm with modified chart values in our AWS Environments. I see the helm charts are provided in this Github repo for multiple istio components.
I want to see in the community how can this be achieved where I can pull the charts and modify them as per my need and deploy the only needed helm charts on our EKS Clusters.

Meaning, how can I install only the istio base & istiod together.

Thanks

**Comment by hanxiaop** [comment, marked as answer]
This may be helpful https://istio.io/latest/docs/setup/install/helm/.
